### DISCUSSION TITLE
Hugging Face部署怎么使用密码

### DISCUSSION THREAD

**Discussion by MC-dusk** [GaiZhenbiao/ChuanhuChatGPT#716] [Q&A]
我开了个public的Hugging Face space以便嵌入自己的网页，但是想设置密码防止被滥用，用[这种方法](https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/%E4%BD%BF%E7%94%A8%E6%95%99%E7%A8%8B#%E8%AE%BE%E7%BD%AE%E7%94%A8%E6%88%B7%E7%99%BB%E5%BD%95%E5%AF%86%E7%A0%81)肯定是不行的，我不知道这种方法是作用于哪种部署，但肯定不是hf，毕竟源码文件都是公开的

**Comment by MC-dusk** [comment]
还有，我看别人api都不显示的，就是空的，为什么我hf里面配置OPENAI_KEY前端key输入框会显示*****

**Comment by Keldos-Li** [comment]
[设置Space](https://github.com/GaiZhenbiao/ChuanhuChatGPT/wiki/%E4%BD%BF%E7%94%A8%E6%95%99%E7%A8%8B#%E8%AE%BE%E7%BD%AEspace)

**Reply by MC-dusk** [reply]
wiki我看过了，但我的问题没解决啊

**Reply by Keldos-Li** [reply]
抱歉，公开space的话，确实没有办法……
